ts/kpt-functions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kpt-functions",
-  "version": "0.14.6-rc.0",
+  "version": "0.14.6",
```

Comment:
Given it's been a while since last release, you may want to release 0.15.0 instead. It's less risky since existing packages will not pick it up automatically.

Reply:
SG. Updated.